### ISSUE TITLE
Having a file simply named Ruby causes the linter to run.

### ISSUE THREAD

**Issue by envygeeks** [AtomLinter/linter-ruby#32]
I think this is bad behavior.  Ruby files have an extension of `.rb` or a `ruby` shebang, but having a plain text file named "ruby" trigger the linter and cause errors all over the place is erratic behavior.

**Comment by Project0** [comment]
I think is a problem with the language-ruby package. The linter triggers on files which are registered by the language package (grammar).

**Comment by envygeeks** [comment]
It seems that you might be right because I just double checked with a file named Ruby and it was flagged as Ruby o.O

**Comment by steelbrain** [comment]
Every time `linter:lint` is called, either internally or via the command or whatever, it gets the scopes of that active text editor's cursor position using something like
```
editor = atom.workspace.getActiveTextEditor()
editor.scopeDescriptorForBufferPosition(editor.getCursorBufferPosition()).scopes
```
and validates all the linters who match that scope and triggers them.

So yes, it's a bug in the scope provider (ie language package), not linter.
